Actions/Deploy/Deploy.ps1

```diff
@@ -120,43 +213,48 @@ else {
             # Continuous deployment is undefined in settings - we will not deploy to production environments
             Write-Host "::Warning::Ignoring environment $($deploymentSettings.EnvironmentName), which is a production environment"
         }
-        elseif ($scope -eq 'Dev') {
-            if (!$sandboxEnvironment) {
-                throw "Scope Dev is only valid for sandbox environments"
-            }
-            $parameters = @{
-                "bcAuthContext" = $bcAuthContext
-                "environment" = $deploymentSettings.EnvironmentName
-                "appFile" = $apps
+        else {
+            if ($dependencies) {
+                InstallOrUpgradeApps -bcAuthContext $bcAuthContext -environment $deploymentSettings.EnvironmentName -Apps $dependencies
```

Comment:
We're not passing anything to the installMode parameter

Reply:
nice catch